example/src/androidTestDebug/java/com/stripe/example/activity/AddFpxPaymentMethodTest.kt

```diff
@@ -60,7 +60,7 @@ class AddFpxPaymentMethodTest {
     private fun launchBankSelector() {
         // launch FPX selection activity
         onView(withId(R.id.examples)).perform(
-            RecyclerViewActions.actionOnItemAtPosition<RecyclerView.ViewHolder>(13, click())
+            RecyclerViewActions.actionOnItemAtPosition<RecyclerView.ViewHolder>(12, click())
```

Comment:
I removed an item from the list above this, so the new position of the Fpx selector is 12